content/webapp/services/prismic/transformers/body.ts

```diff
@@ -132,6 +111,7 @@ function transformMediaObjectListSlice(slice: MediaObjectListSlice): BodySlice {
               image: transformImage(mediaObject.image)!,
             };
           }
+          return undefined;
```

Comment:
linting fix: `Array.prototype.map() expects a value to be returned at the end of arrow function`